modules/fetch_seqs.nf

```diff
@@ -13,66 +13,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** 1) Add the fetching stage to the workflow
- * 
- * This first stage of the workflow aims to fetch all the reference mtDNA sequences (RefSeq) 
- * in GenBank for the tick genus Ixodes
- * 
- * We analyse the sequences downloaded to get basic stats using the statistics function. 
- * It returns the total number of sequences stored and the arithmetic mean of the array elements of the sequences, 
- * standard deviation of the length of the sequences, minimum and maximum values of their length.
- * 
- * Finally, all the information can be saved in a file with the SeqRecord objects in a GENBANK format
-**/
-
-nextflow.enable.dsl = 2
-
 process FETCH_SEQS {
-
-    publishDir "./data/${name}/fetch", mode: 'copy', overwrite: false
+    publishDir "${params.output_dir}/fetch", mode: 'copy', overwrite: false
 
     input:
-    val total_query
-    val name
+        val query
 
     output:
-    path "${name}.gb"
+        tuple path('output.gb'), val('genbank')
```

Comment:
Most of the other methods require to provide both the output file and its format for the next step to work. Since we are "reusing channels" if steps are skipped, these two elements need to be provided by every module for consistency.